### DISCUSSION TITLE
Using `OrbitControls` with `PerspectiveCamera`

### DISCUSSION THREAD

**Discussion by chrisjsewell** [pmndrs/drei#406] [Q&A]
Apologies if this is a newb question, but I can't seem to get `OrbitControls` to work with `PerspectiveCamera`. Take a simple scene like:

```jsx
<Canvas>

    <OrbitControls />

    <ambientLight />
    <pointLight position={[10, 10, 10]} />

    <Box />

</Canvas>
```

(which works fine)

I would like to add a `PerspectiveCamera`, in the first instance to change the FoV.
However, from https://github.com/pmndrs/drei#perspectivecamera and https://github.com/pmndrs/drei#controls, it is unclear to me what the hierarchy should be to add a `PerspectiveCamera`?
Everything I try seems to deactivate the controls.
`

**Comment by drcmda** [comment]
You can add the camera where ever you want. If you give it a makeDefault prop it's gonna be the default camera.
